src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/DocumentType.java

```diff
@@ -12,6 +12,8 @@
     DIRECTION_NOTICE("Direction Notice", "Directions Notice"),
     DL6("dl6", "DL6"),
     DL16("dl16", "DL16"),
+    D609_97("609-97"),
```

Comment:
Are these 2 document types needed in the common library? 

If so, you might want to use an id that doesn't have a dash in it:
```
   D609_97("60997", "609-97"),
   D609_98('60998", "609-98"),
```
This should work, just in case it ever gets modelled as a Java field name

Reply:
I have removed the document type time being, I will add this when evidence share api ready for push. As this need in evidence share,